doc/udisks2-sections.txt.in.in

```diff
@@ -1476,6 +1520,9 @@ udisks_partition_table_call_create_partition
 udisks_partition_table_call_create_partition_finish
 udisks_partition_table_call_create_partition_sync
 udisks_partition_table_complete_create_partition
+udisks_partition_table_get_partitions
+udisks_partition_table_dup_partitions
+udisks_partition_table_set_partitions
```

Comment:
Are we really supporting ``udisks_partition_table_set_partitions()``?

Reply:
It can be used from udisks code and we generate documentation like this for all properties.
http://storaged.org/doc/udisks2-api/latest/UDisksPartitionTable.html#udisks-partition-table-set-type-

Reply:
Looks like we should mark certain properties as RO

Reply:
All our properties are marked like that:

`<property name="Partitions" type="ao" access="read"/>`

Reply:
Hmm, then it looks like the gdbus-codegen is doing something wrong. Anyway, that's not something we should be aiming for in the PR.